app/scripts/views/sign_in_bounced.js

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Notifies the user that their sign-in confirmation email bounced.
+ */
+define(function (require, exports, module) {
+  'use strict';
+
+  const BackMixin = require('views/mixins/back-mixin');
+  const BaseView = require('views/base');
+  const Cocktail = require('cocktail');
+  const FlowEventsMixin = require('views/mixins/flow-events-mixin');
+  const Session = require('lib/session');
+  const Template = require('stache!templates/sign_in_bounced');
+
+  const SignInBouncedView = BaseView.extend({
+    events: {
+      'click #create-account': '_createAccount'
+    },
+
+    template: Template,
+
+    initialize (options) {
+      this._formPrefill = options.formPrefill;
+    },
+
+    setInitialContext (context) {
+      context.set({
+        email: this.model.get('email'),
+        escapedSupportLinkAttrs: 'id="support" href="https://support.mozilla.org/" target="_blank" data-flow-event="link.support"'
```

Comment:
Obviously I'll update the flow event docs if/when this gets merged. Updating them ahead of time seems unnecessary.